applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -519,7 +526,7 @@ def _get_element_condition_replace_settings(self):
                     self.settings["element_replace_settings"]["element_name"].SetString("EulerianConvDiff3D")
                 else:
                     self.settings["element_replace_settings"]["element_name"].SetString("EulerianConvDiff3D8N")
-        elif element_name in ("LaplacianElement","AdjointHeatDiffusionElement"):
+        elif element_name in ("LaplacianElement","AdjointHeatDiffusionElement","SymbolicQSConvectionDiffusionExplicit","SymbolicDConvectionDiffusionExplicit"):
```

Comment:
I'd remove `Symbolic` keyword from the element name. We will do so in the FluidDynamicsApplication soon.

Reply:
Done.